scripts/metabiome.sh

```diff
@@ -13,6 +13,18 @@ function usage() {
 cat<<HELP_USAGE
 Usage: metabiome [Commands|Options]
 
+
+							Welcome to:
+
+			███╗   ███╗███████╗████████╗ █████╗ ██████╗ ██╗ ██████╗ ███╗   ███╗███████╗
+			████╗ ████║██╔════╝╚══██╔══╝██╔══██╗██╔══██╗██║██╔═══██╗████╗ ████║██╔════╝
+			██╔████╔██║█████╗     ██║   ███████║██████╔╝██║██║   ██║██╔████╔██║█████╗
+			██║╚██╔╝██║██╔══╝     ██║   ██╔══██║██╔══██╗██║██║   ██║██║╚██╔╝██║██╔══╝
+			██║ ╚═╝ ██║███████╗   ██║   ██║  ██║██████╔╝██║╚██████╔╝██║ ╚═╝ ██║███████╗
+			╚═╝     ╚═╝╚══════╝   ╚═╝   ╚═╝  ╚═╝╚═════╝ ╚═╝ ╚═════╝ ╚═╝     ╚═╝╚══════╝
+
+				A flexible and modular pipeline for metagenomic analysis!!!
+
```

Comment:
Tabs should be replaced with spaces. Line length should be of max 80 chars, as it does not look fine on terminals with default size:
![Selection_001](https://user-images.githubusercontent.com/63323077/120421558-24e69280-c32c-11eb-9103-22b6660e682b.png)